src/components/docs-search.js

```diff
@@ -1,12 +1,15 @@
 import React, { useEffect } from "react"
 import { navigate } from "@reach/router"
+import { graphql, useStaticQuery } from "gatsby"
 
 import Helmet from "react-helmet"
 
 import DocsTitle from "./docs-title"
 import AccessibleSVG from "./accessible-svg"
 
-const DocsSearch = () => {
+const DEFAULT_PATH_PREFIX = process.env.NODE_ENV !== "production" ? "/workers" : ""
```

Comment:
We should not be hard-coding `"/workers"` anywhere inside `src/components/**.js`.

Reply:
Also why doesn’t production have a default of `""`? It seems like this may also warrant a comment explaining the difference in behavior.

Reply:
i'm actually not sure what problem this solves, lol. i guess i needed it at some point last night, but now i'm finding that things work just fine without it!